atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -28,7 +28,7 @@
 
 public abstract class AbstractTransactionManager implements TransactionManager {
     public static final Logger log = LoggerFactory.getLogger(AbstractTransactionManager.class);
-
+    protected final TimestampCache timestampCache = new TimestampCache();
```

Comment:
nit: change name to make it more clear what it contains, possibly transactionTimestampCache?  commitTimestampCache?